### ISSUE TITLE
rebin short-cadence poster-child data and re-run

### ISSUE THREAD

**Issue by davidwhogg** [davidwhogg/HoneyComb#22]
Can we take the short-cadence data on the poster child, and:
- bin 2x2

(meaning: average adjacent pairs of data points)
- run SPG, LS, FFT

and then repeat -- ie, rebin again and do it again, to see how the signals degrade as we rebin?

**Comment by RuthAngus** [comment]
Sure - it takes a while to run but I'll update you when I have plots.

**Comment by RuthAngus** [comment]
Here are some rebinning plots for superpgram (top), fft (middle) and Lomb-Scargle (bottom). I am a little confused about why the superpgram and fft versions look so similar, but the LS version looks different. I would have expected LS and spg to look more same-y. Does this make sense to you?

spg:
![binning_comparison_spg](https://cloud.githubusercontent.com/assets/2834161/6993941/2533d276-dad4-11e4-91d6-7c40927d90e2.png)
fft:
![binning_comparison_fft](https://cloud.githubusercontent.com/assets/2834161/6993943/2c008932-dad4-11e4-905e-955285bc1b2b.png)
LS:
![binning_comparison_ls](https://cloud.githubusercontent.com/assets/2834161/6993945/3118218c-dad4-11e4-8c32-640b781d508e.png)

And I have another question... in the above spg plot I'm not taking the binning into account when I compute the footprint. I leave the delta t in the footprint at the short cadence interval of ~1 min. When I _do_ take the changing time interval into account (i.e. doubling it for 2 points-per-bin, etc) I get the following plot. I am confused though because I would have expected this to look better than the above example. Am I misunderstanding something? Does this make sense to you, or is it more likely that I've got a bug somewhere? 

![binning_comparison](https://cloud.githubusercontent.com/assets/2834161/6993969/27c87752-dad5-11e4-99b2-8aa4cb08a161.png)

**Comment by davidwhogg** [comment]
No, I don't understand this.  Let's try a simpler thing out of the gate:   Let's make Fake data that _exactly_ conforms to the SPG model...  New issue coming while this sits on hold
